ci/docker/concourse-runner/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apk add --no-cache \
   maven \
   tini \
   docker-compose>1.25.4-r1 \
+  git \
```

Comment:
It doesn't matter too much, but git is available from the alpine main repo as well as edge. This will install a bleeding edge version.